vue-components/stories/TextArea.stories.ts

```diff
@@ -73,12 +87,17 @@ basic.argTypes = {
 };
 
 export function all(): Component {
-	return {
-		components: { TextArea },
-		template: `
-			<div style="max-width: 95%">
-				<TextArea label="Label" placeholder="Placeholder" />
-			</div>
-		`,
-	};
+    return {
+        components: { TextArea },
+        template: `
+            <div>
+                <div style="max-width: 95%;">
+                    <TextArea label="Default" placeholder="Placeholder" />
+                </div>
+                <div style="max-width: 95%; margin-top: 1em;">
+                    <TextArea label="Read Only" placeholder="Placeholder" :read-only="true" value="Potatoes\nArtichokes" />
```

Comment:
```suggestion
                    <TextArea label="Read Only" placeholder="Placeholder" :read-only="true" value="Content within a read-only text area can be selected, but not edited." />
```

Reply:
Mind if we don't eat our greens today?

Reply:
Not at all, bring in the cheeseburgers :hamburger: